### PR TITLE
Differentiate "geometry" from "physics" materials in ImportData

### PR DESCRIPTION
**Pull request by sethrj**
This differentiates between true material compositions associated with the geometry, and "physics" materials which are modified by physics cutoffs inside a region that's applied to a particular volume. The "geometry materials" are equivalent to G4Material, and the "physics material" are the G4MaterialCutsCouple which is what we use in Celeritas (and need to generalize for #907, #983).

### CODE REVIEW

File: src/celeritas/io/ImportParameters.hh [hunk 1]
```diff
@@ -17,6 +17,24 @@
...
 {
+//---------------------------------------------------------------------------//
+/*!
+ * User-imposed cutoffs and restrictions.
+ *
+ * Used as tracking cuts, see \c G4UserLimits , used by \c G4UserSpecialCuts.
+ * As "production" cuts, only for electron/positron/proton/gamma, see \c
+ * G4ProductionCuts .
+ */
+struct ImportCutoff
```
Comment (stognini) [inline src/celeritas/io/ImportParameters.hh]: I'm pondering about this name. This is another set of cuts beyond the `ImportProductionCuts` which originally are loaded into our `CutoffParams`---which bears a strikingly similar name to this one. So maybe we need that to be `ProductionCutParams` instead?

Comment (sethrj) [reply src/celeritas/io/ImportParameters.hh]: I guess these are tracking cuts; I think we should combine them with production cuts in the codebase. I've been trying to untangle the particle production cuts, regions, "cut couple", etc. I forgot that I didn't actually implement the reading for these cuts from G4UserLimits .... I'll just remove them for this PR and we can add them to an "extended cut" set later. We should also determine how to make the `apply_cuts` and `lowest_electron_energy` EM parameters consistent with these options.

Comment (stognini) [reply src/celeritas/io/ImportParameters.hh]: Yep, you reached another question I had while going over it: that this `ImportCutoff` struct is not being used yet. Okay, that sounds good to me. And later on, I think it makes sense to keep a `CutoffParams` that encompasses both production cuts and extended cuts by adding both to the `CutoffParamsData`.

File: src/celeritas/ext/GeantImporter.cc [hunk 2]
```diff
@@ -709,38 +755,45 @@ import_materials(GeantImporter::DataSelection::Flags particle_flags)
...
+    CELER_ENSURE(!materials.empty());
+    return materials;
+}
 
-            // Add material's element information
-            material.elements.push_back(elem_comp);
-        }
+//---------------------------------------------------------------------------//
+/*!
+ * Return a populated \c ImportVolume vector.
```
Comment (amandalund) [inline src/celeritas/ext/GeantImporter.cc]:
```suggestion
 * Return a populated \c ImportRegion vector.
```

File: app/celer-dump-data.cc [hunk 3]
```diff
@@ -55,8 +55,8 @@ void print_particles(ParticleParams const& particles)
     cout << R"gfm(
 # Particles
 
-| Name              | PDG Code    | Mass [MeV] | Charge [e] | Decay [1/s] |
-| ----------------- | ----------- | ---------- | ---------- | ----------- |
+| Name              | PDG Code    | Mass [MeV] | Charge [e] | Decay [1/time] |
+| ----------------- | ----------- | ---------- | ---------- | -------------- |
```
Comment (amandalund) [inline app/celer-dump-data.cc:60]:
```diff
-             << setw(11) << setprecision(3) << p.decay_constant()
+             << setw(14) << setprecision(3) << p.decay_constant()
```

File: app/celer-dump-data.cc [hunk 4]
```diff
@@ -176,19 +196,29 @@ void print_materials(std::vector<ImportMaterial>& materials,
...
     //// PRINT CUTOFF LIST ///
 
     cout << R"gfm(
-## Cutoffs
+# Material physics
 
-| Material                   | Particle  | Energy [MeV] | Range [cm] |
-| -------------------------- | --------- | ------------ | ---------- |
+| Material ID/Name                | Particle  | Energy [MeV] | Range [len] |
+| ------------------------------- | --------- | ------------ | ----------- |
```
Comment (amandalund) [inline app/celer-dump-data.cc:219]:
```diff
-                 << setw(12) << cuts.energy << " | " << setw(10) << cuts.range
+                 << setw(12) << cuts.energy << " | " << setw(11) << cuts.range
```

**Review by amandalund — APPROVED** [review]
Looks good to me!

**Review by stognini — APPROVED** [review]

**Comment by stognini** [comment]
Looks great. Thanks!

**Pull request merged**